fredknowsit/jquery-expert
# Cesc/dev 1511/bugfix/bubble click duplicates start message

## Context
Problem: Opening and closing the beacon multiple times, shows duplicate messages to the user.

### Description
This pull request aims to solve this situation.

## Status
READY


## Steps to Test
1. Open the beacon
2. Close the beacon
3. Repeat 1. and 2
4. *Output*: No duplicates messages shall appear
